packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -3567,6 +3567,78 @@ Expected input body:
 { "property": "hello" }
 ```
 
+### Type_Union_receiveFirstNamedUnionValue
+
+- Endpoint: `post /type/union/receive/model1`
+
+This test is tesing receiving the first union value in named union property.
+
+Expect request:
+
+```json
+{ "kind": "model1" }
+```
+
+Expect response:
+
+```json
+{ "namedUnion": { "name": "model1", "prop1": 1 } }
+```
+
+### Type_Union_receiveIntArray
+
+- Endpoint: `post /type/union/receive/int-array`
+
+This test is tesing receiving an int array value in simple union property.
+
+Expect request:
+
+```json
+{ "kind": "int-array" }
+```
+
+Expect response:
+
+```json
+{ "simpleUnion": [1, 2] }
+```
+
+### Type_Union_receiveSecondNamedUnionValue
```

Comment:
```suggestion
### Type_Union_receiveSecondTypeOfNamedUnionValue
```

Reply:
I would like the name to keep alignment to the existing cases: https://github.com/Azure/cadl-ranch/blob/a71bd94f358a3ca8e276e2345a87b023b05eda85/packages/cadl-ranch-specs/http/type/union/mockapi.ts#L20
Unless I could change them both otherwise I rather keep these names.